ui/src/router/components/form/components/ensembler_config/typeOptions.js

```diff
@@ -1,69 +1,99 @@
 import React, { Fragment } from "react";
 
-const typeOptions = [
-  {
-    value: "nop",
-    inputDisplay: "None",
-    description: (
-      <Fragment>
-        Turing will simply return response from the route marked as the{" "}
-        <strong>Final Response</strong>
-      </Fragment>
-    ),
-  },
-  {
-    value: "standard",
-    inputDisplay: "Standard",
-    description: (
-      <Fragment>
-        Turing will select the response from one of the routes, based on the
-        configured mapping between routes and experiment treatments
-      </Fragment>
-    ),
-  },
-  {
-    value: "docker",
-    inputDisplay: "Docker",
-    description: (
-      <Fragment>
-        Turing will deploy specified image as a post-processor and will send to
-        it responses from all routes, together with the treatment configuration,
-        for the ensembling
-      </Fragment>
-    ),
-  },
-  {
-    value: "pyfunc",
-    inputDisplay: "Pyfunc",
-    description: (
-      <Fragment>
-        Turing will build and deploy the selected pyfunc ensembler and will send
-        to it responses from all routes, together with the treatment
-        configuration, for ensembling
-      </Fragment>
-    ),
-  },
-  {
-    value: "external",
-    inputDisplay: "External (Coming Soon)",
-    description: (
-      <Fragment>
-        Turing will send responses from all routes, together with treatment
-        configuration, to the external URL for ensembling
-      </Fragment>
-    ),
-    disabled: true,
-  },
-];
+const typeOptions = {
+  HTTP_JSON: [
+    {
+      value: "nop",
+      inputDisplay: "None",
+      description: (
+        <Fragment>
+          Turing will simply return response from the route marked as the{" "}
+          <strong>Final Response</strong>
+        </Fragment>
+      ),
+    },
+    {
+      value: "standard",
+      inputDisplay: "Standard",
+      description: (
+        <Fragment>
+          Turing will select the response from one of the routes, based on the
+          configured mapping between routes and experiment treatments
+        </Fragment>
+      ),
+    },
+    {
+      value: "docker",
+      inputDisplay: "Docker",
+      description: (
+        <Fragment>
+          Turing will deploy specified image as a post-processor and will send
+          to it responses from all routes, together with the treatment
+          configuration, for the ensembling
+        </Fragment>
+      ),
+    },
+    {
+      value: "pyfunc",
+      inputDisplay: "Pyfunc",
+      description: (
+        <Fragment>
+          Turing will build and deploy the selected pyfunc ensembler and will
+          send to it responses from all routes, together with the treatment
+          configuration, for ensembling
```

Comment:
nit: `and send the responses from all routes..., for ensembling the final response.`